submittedResponses.tf

```diff
@@ -19,3 +19,7 @@ resource "aws_dynamodb_table" "submitted-responses-table" {
     Environment = "${var.env}"
   }
 }
+
+output "submitted_responses_table_name" {
+  value = "${aws_dynamodb_table.submitted-responses-table.name}"
```

Comment:
- The resource name should be `submitted_responsestable` rather than `submitted-responses-table`
- The filename should be submitted_responses.tf